### ISSUE TITLE
ops/set up MIS report

### ISSUE THREAD

**Issue by chris-dare** [serenity-health/roadmap#17]
**Tell us about your request!**
Doctors need a report to view details of encounters with patients. This is currently referred to as the "MIS report"
@DanielDarko12 has shared a sample of this report with @dexios1

**Which app does this concern?**
Provider portal

**What will be the benefit of this feature?**
I'll refrain from assuming and get write what @DanielDarko12 describes

**Describe the solution you'd like**
Create a report via metabase. This report will be tabular and contain the following fields:

1. Date
2. MR No. 
3. Age 
4. DOB 
5. Sex 
6. Registration Date 
7. Visit Type 
8. Doctor Name 
9. Internal Referal 
10. Doctor Refer To 
11. Specialty Refered To 
12. BP 
13. Pulse 
14. Resp.
15. Temp. 
16. Height
17. Weight 
18. Doctor Speciality 
19. Laboratory Requested 
20. Redialogy Requested
21. Pharmacy Requested 
22. No of Medicine Prescribed 
23. Provisional Diagnosis 
24. Final Diagnosis 
25. Covid Like Symptoms 
26. Traveling
27. Patient name

Filter fields:
We're yet to discuss these

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by DanielDarko12** [comment]
Yes, we can drop covid like symptoms and travelling and make room for other parameters to be accessible later if needed

Daniel Osafo Darko , MD, MGCPS(Fam Med)Specialist Family Physician+233 50 644 32 74http://www.linkedin.com/in/daniel-osafo-darko-592baa87
 "Let your life reflect the faith you have in God. Fear nothing and pray about everything. Be strong, trust God’s word, and trust the process." - Germany Kent

 

    On Wednesday, January 5, 2022, 01:06:20 PM GMT, Chris Dare ***@***.***> wrote:  
 
 


Tell us about your request!
Doctors need a report to view details of encounters with patients. This is currently referred to as the "MIS report"
@DanielDarko12 has shared a sample of this report with @dexios1

Which app does this concern?
Provider portal

What will be the benefit of this feature?
I'll refrain from assuming and get write what @DanielDarko12 describes

Describe the solution you'd like
Create a report via metabase. This report will be tabular and contain the following fields:
   
   - Date
   - MR No.
   - Age
   - DOB
   - Sex
   - Registration Date
   - Visit Type
   - Doctor Name
   - Internal Referal
   - Doctor Refer To
   - Specialty Refered To
   - BP
   - Pulse
   - Resp.
   - Temp.
   - Height
   - Weight
   - Doctor Speciality
   - Laboratory Requested
   - Redialogy Requested
   - Pharmacy Requested
   - No of Medicine Prescribed
   - Provisional Diagnosis
   - Final Diagnosis
   - Covid Like Symptoms
   - Traveling

Filter fields:
We're yet to discuss these

Describe alternatives you've considered
N/A

Additional context
N/A

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by chris-dare** [comment]
KPIs generated from this data for medical:
1. Top 10 diseases
2. Top 10 reasons for OPD visits at provider (bar chart)
3. Tell top 10 diseases by location
4. Top 10 reasons for OPD visits at provider (bar chart)
5. Top 10 lab requested
6. Top 10 medication requests
** Usually generated monthly and annually
Helpful filters: Specialty and revenue tags (revenue tags yet to be built it)

Primary care questions:
1. What percentage of patients are hypertensive and had BP below 140/90? 
2. What percentage of diabetics are controlled?
Control definition: FBS of less than 6.5 or HBA1c less than 7%
3. What percentage of diabetic patients have had an HBA1c done in the last 6 months?
This is a global standard evaluation for managing diabetics. Evaluating compliance to medical protocols
4. What percentage of patients diagnosed with malaria had a malaria test done?
This is a WHO and Ghana health service requirement for evidence based management before a medication/treatment plan
5. What percentage of patients diagnosed with UTI (urinary tract infection) had a urine test done?
This is a global standard for an antibiotic stewardship
6. What percentage of patients diagnosed with anemia/anaemia?
7. What percentage of patients diagnosed with pneumonia had a chest xray/CT chest done?
** Groupings:
- Grouped by GP and specialists
- Grouped by week and month
- Grouped by location/site
8. Doctor to patient ratio questions (daily):
- Provider ratio: 1: 4 patients per hour
- Provider 1: 44 patients per 12 hour shift
9. Total PCR done
10. COVID test positivity trends
11. How many referrals have been done? Group by internal, external and specialty
12. How many patients did each doctor see?
13. How much revenue did each doctor generate from their consultations with patients? (cost of consultations only)

Mortality analysis questions
TBD: Revisit with Dr. Darko.

**Comment by chris-dare** [comment]
Health hub interview

Summary of meeting with Lauretta Elloh-Donkor:
1. Get a list of labs requested:
a. Patient's first, other, last name
b. MR number
c. Lab requested
d. Mobile number
e. Date of visit (when requested)
f. Doctor that saw them

This is for the following tests:
1. Fasting blood sugar -> this has been setup
2. Glycated hemoglobin (HBA1c) -> this has also been set up
3. Random blood sugar -> couldn't find results under this test. Will need to speak with Dr. Ben and diagnostics team to confirm that this test has been set up

Action points:
1. Confirm set up of random blood sugar test
2. Get a sign off from Nana Pokua on sharing the chronic disease data with Lauretta.

**Comment by chris-dare** [comment]
## Finance requirements for MIS report:

### Billing dataset parameters:
1. Receipt ID
2. Service or product name
3. Service category
4. Service request category
5. Payment method
6. Charge
7. Status (filtered by billed and billable)
8. Site name
9. Date of creation
10. Payer name
11. Practitioner name
12. Service specialty
13. Cashier name
Status: Done

### What questions do the finance team try to answer:
1. What is the total revenue generated for the month? (done)
2. What is the total revenue generated for the month by payment method? (done)
3. What is the total revenue generated for the month per business unit (revenue tag)? (revenue tags don't exist yet. Will provide the billing dataset for this task)
4. What is the total revenue generated for the month by site? (done)
5. What is the total revenue generated for the month per clinical program (specialty)? - (Done - Service name was used as proxy)
7. What's the footfall per service category? (done)
8. What's the billing footfall per day? (done)
9. What's the footfall per week? (done)
8. What's the footfall per business unit/revenue center? (currently not supported. Will provide billing dataset for this task)
9. What's the total revenue per payer per month? (done)
10. What is the total revenue per doctor? i.e. Specialist payroll (done)
This involves their consulations performed by patients as well as orders they make
In the case where 2 doctors see a patient during the consultation, what happens?
11. What are the confirmed pharmacy bills? (done)
12. What are the unconfirmed pharmacy bills? (done)
--- Additional requirements from 1 March 2022 meeting with Finance team
13. How much cash has been received by each cashier per day, week and month by site? (includes copayments)
Cashiers need access to a separate dashboard/report for their daily recon

Dates to be used: Transaction date and not billing date
Definition of cashier: Practitioner who received payment and not practitioner who raised bill

Important to ensure that doctors make service requests on in the app instead of on paper to allocate revenue to them


A dashboard has been created for this and will be sent on the 1st calendar day of each month at 1am. A copy of the metrics/report for the previous month has been sent as well?
Philip and Dorothy will review and revert on the information provided. Modifications will be made if required.

*Update from 14 March 2022 conversation with Philip, Dorothy and Kofi*
Main concerns have to do with balancing the books for cash transactions
- Need to transfer transaction dates to new field for payment dates
- Need to confirm bill cancelation process - whether or not to include bills for which cancelations were requested by not approved
- Important to review billing dashboard
- Replace EOD day revenue reports with new revenue report pivot table with filters for cashier name, revenue tags and payment date

**Comment by chris-dare** [comment]
Notes from CE interview:

1. Retrieve encounter patient list: Which patients did a doctor see for a given period?

**Comment by chris-dare** [comment]
Note: Lost metabase app data on Friday. App was not configured with backups. 
Set up BI on metabase cloud (has backups built in) to mitigate risk of loss. Rebuilding dashboard between Mon 25 - Wed 27 2022

BI report recovery to deliver the following by 25 April 22 COB:
1. Billing dashboard
2. CE dashboard
3. Services report
4. Service price tiers report
5. Pharmacy inventory report
6. Pharmacy inventory adjustment report
7. Claims report
8. Revenue report

**Comment by chris-dare** [comment]
@rejoicehormeku

Here is the link to diagnostics reporting: https://github.com/serenity-health/roadmap/issues/81

**Comment by chris-dare** [comment]
Here's the link to the customer experience report requirement: #18
